kubeflow-components/overlays/osc/osc-cl1/kustomization.yaml

```diff
@@ -1,5 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
This is actually deployed on the `osc-cl1` cluster but not managed by ArgoCD. I think it's fine to remove the `kubeflow-components` dir, but can you move the kfdef this is referencing base, to: https://github.com/operate-first/apps/tree/master/kfdefs/overlays/osc/osc-cl1 under a new directory called `kubeflow` and include it in this kustomization as well: https://github.com/operate-first/apps/blob/master/kfdefs/overlays/osc/osc-cl1/kustomization.yaml

Reply:
Moved kfdef as requested :+1: .